### PR TITLE
more on the aur, 'miscellaneous' fuck-ups

### PR DESCRIPTION
**Pull request by tofilwiktor**

### CODE REVIEW

File: src/routes/index.svelte.md [hunk 1]
```diff
@@ -61,7 +65,8 @@ run install scripts for anything sure sounds neat, right?
...
+That, and Manjaro *doesn't actually support* the AUR. Despite their [contradictory messages](https://web.archive.org/web/20220221092555/https://forum.endeavouros.com/t/is-aur-down-again/24287/9), Manjaro hides behind [blaming the users of pamac](https://web.archive.org/web/20220221090752/https://forum.manjaro.org/t/aur-please-restrain-yourself/103318). They provide insufficent warnings about the AUR and the potential risks, while providing a simplified interface for installing AUR packages via pamac.  This is akin to letting someone with no briefing into a construction site. Sure, the heavy machinery might be quicker than using a shovel, but they are ultimately putting themselves in danger due to not being made aware of the consequences. The AUR is insecure and you need to inspect the PKGBUILD before building. Blindly installing AUR packages can be harmful to your system.
```
Comment (EmeraldSnorlax) [inline src/routes/index.svelte.md:68]: this might be better placed in the opening paragraph above, and also explaining that the aur contains user scripts that you should skimread before running rather than the vague statement that it can be harmful

File: src/routes/index.svelte.md [hunk 2]
```diff
@@ -43,6 +43,10 @@ https://web.archive.org/web/20150409040851/https://manjaro.github.io/expired_SSL
 
 Holding back packages for two weeks can also cause security issues, but this issue is probably better addressed in the stability section.
 
+## "Suitable for beginners"
+
+Manjaro claims that it is a distribution that's suitable for both beginners and experienced users, this is also an opinion you can often hear on the internet. In reality, beginners shouldn't use rolling-release distros. If you're using one you need to be prepared for potential issues and bugs and know how to troubleshoot them because of the very frequent updates - with rolling-release distros you get the latest versions of packages which, while it might sound good, may result in problems with the packages. Frequent kernel updates can sometimes result in issues with drivers, and you have to troubleshoot these issues as well.
```
Comment (EmeraldSnorlax) [inline src/routes/index.svelte.md:48]: i'm skeptical of the idea beginners shouldn't use rolling distros -- rolling just means there are no set release candidates with an EoL. bugs are also possible with upgrading packages in leap style distros. its more important how well tested everything is and how close to the edge a distro rides -- thats more on the distro's individual protocol rather than inherent to rolling as a whole (see openSUSE tumbleweed for an example of what i'd consider a stable rolling distro). i'm open to discussion on this though, perhaps the wording could be improved to better reflect this?

File: src/routes/index.svelte.md [hunk 3]
```diff
@@ -101,6 +106,10 @@ blocked again. This may have been the cause for the day's earlier outage.
 
 While these incidents were in no way intentional, it highlights the poor QA testing that Manjaro performs. This has happened on two separate occasions in less than two years.
 
+## Miscellaneous
+
+[Their system update script runs `rm` on the lockfile mid-transaction.](https://gitlab.manjaro.org/packages/core/manjaro-system/blob/3b806753e245b7ec7e18bb674e916e28d751a429/manjaro-update-system.sh#L45(https://archive.fo/dofw8)).
```
Comment (EmeraldSnorlax) [inline src/routes/index.svelte.md:111]: a little more context on this can be given -- maybe something like *why* this is bad

**Review by EmeraldSnorlax — CHANGES_REQUESTED** [review]
im curious to know what you and others think of this review -- for the rolling claim, im not sure it belongs in this article

**Comment by EmeraldSnorlax** [comment]
could you turn on maintainer edits?

File: src/routes/index.svelte.md [hunk 4]
```diff
@@ -101,6 +102,10 @@ blocked again. This may have been the cause for the day's earlier outage.
 
 While these incidents were in no way intentional, it highlights the poor QA testing that Manjaro performs. This has happened on two separate occasions in less than two years.
 
+## Miscellaneous
+
+[Their system update script runs `rm` on the lockfile mid-transaction.](https://gitlab.manjaro.org/packages/core/manjaro-system/blob/3b806753e245b7ec7e18bb674e916e28d751a429/manjaro-update-system.sh#L45(https://archive.fo/dofw8)) The lockfile is in place to prevent multiple instances of pacman from trying to alter the package database at the same time. Sometimes, when pacman is interrupted the stale lockfile can remain, in those cases removing the lockfile is a common troubleshooting step. However you should only do that when you are **absolutely certain** there are no other pacman instances running and Manjaro's script does this preventively, without even prompting the user.
```
Comment (EmeraldSnorlax) [inline src/routes/index.svelte.md:107]: i've done a little more digging and it seems like this is an old issue that was fixed a while ago (it's no longer [in current versions](https://gitlab.manjaro.org/packages/core/manjaro-system/-/blob/master/manjaro-update-system.sh))

i dont know if this should be left out, or include it with the context of "they've fucked this up in the past"

**Comment by bddvlpr** [comment]
meow 👻

**Comment by tofilwiktor** [comment]
@EmeraldSnorlax I do have maintainer edits turned on
